tests/unit/constraints/test_tabular.py

```diff
@@ -50,9 +63,239 @@ def test___init__(self):
         )
 
         # Assert
-        assert instance.transform == dummy_transform
-        assert instance.reverse_transform == dummy_reverse_transform
-        assert instance.is_valid == dummy_is_valid
+        assert instance._transform == dummy_transform
+        assert instance._reverse_transform == dummy_reverse_transform
+        assert instance._is_valid == dummy_is_valid
+
+    def test__run_transform_table(self):
+        """Test the ``CustomConstraint._run`` method.
+
+        The ``_run`` method excutes ``transform`` and ``reverse_transform``
+        based on the signature of the functions.
+
+        Setup:
+        - Create dummy functions of general type.
```

Comment:
I see this on most of these. What does create functions of a certain type mean? Can this be slightly more descriptive?